assets/app/features/upgrade/controllers/upgrade-administration-repositories-checks.controller.js

```diff
@@ -67,20 +68,24 @@
                         });
                         // Iterate over the checks to determine the validity of the step
                         vm.repoChecks.valid = Object.keys(vm.repoChecks.checks).every(function(k) {
+                            if (vm.repoChecks.checks[k].status === false) {
+                                repoErrors.push(vm.repoChecks.checks[k].label)
+                            }
                             return vm.repoChecks.checks[k].status === true
                         });
 
                         // if all the repos are ok, complete the step
                         if (vm.repoChecks.valid) {
                             upgradeStepsFactory.setCurrentStepCompleted();
+                        } else {
+                            // TODO(itxaka): This is a tmp thing, title and style should be clarified
+                            vm.error = { title: 'Some repositories are not ready', body: repoErrors.join(', ') };
```

Comment:
please move those texts to translations file (e.g. like here: https://github.com/crowbar/crowbar-ui/pull/115)